src/redux/books/books.js

```diff
@@ -0,0 +1,35 @@
+const ADD = 'bookstore/books/ADD';
+const REMOVE = 'bookstore/books/REMOVE';
+const INITIAL_STATE = [];
+
+export function addBook(book) {
+  return {
+    type: ADD,
+    payload: book,
+  };
+}
+
+export function removeBook(id) {
+  return {
+    type: REMOVE,
+    payload: id,
+  };
+}
+
+export default function reducer(state = INITIAL_STATE, action) {
+  switch (action.type) {
+    case ADD:
+      return [
+        ...state,
+        {
+          id: 1,
+          title: action.payload.title,
+          author: action.payload.author,
+        },
+      ];
```

Comment:
- [ ] Please implement the changes requested by the previous reviewer. They provided a good hint for the solution, kindly use it

Reply:
thanks, done.